jobs/garden/templates/garden_ctl.erb

```diff
@@ -1,5 +1,18 @@
 #!/bin/bash -e
 
+<% if_p('env.http_proxy') do |http_proxy| %>
+	export HTTP_PROXY=<%= http_proxy %>
+	export http_proxy=<%= http_proxy %>
+<% end %>
+<% if_p('env.https_proxy') do |https_proxy| %>
+	export HTTPS_PROXY=<%= https_proxy %>
+	export https_proxy=<%= https_proxy %>
+<% end %>
+<% if_p('env.no_proxy') do |no_proxy| %>
+	export NO_PROXY=<%= no_proxy %>
+	export no_proxy=<%= no_proxy %>
+<% end %>
```

Comment:
Is there a reason we need to set both the uppercase and lowercase variants of each of these?

Also, I think it's idiomatic for a BOSH ctl script to have things like the `RUN_DIR` and `PIDFILE` declarations right at the top, so this would be better placed below that initial block of env setup.